tests/queries/0_stateless/03172_error_log_table_not_empty.sql

```diff
@@ -0,0 +1,25 @@
+-- Throw three random errors: 111, 222 and 333
+SELECT throwIf(true, 'error_log', toInt16(111)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 111 }
+SELECT throwIf(true, 'error_log', toInt16(222)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 222 }
+SELECT throwIf(true, 'error_log', toInt16(333)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 333 }
+
+-- Wait for more than collect_interval_milliseconds to ensure system.error_log is flushed
+SELECT sleep(2) FORMAT NULL;
```

Comment:
`SYSTEM FLUSH LOGS` should guarantee that logs are flushed.

Reply:
That's making sure the logs are flushed for whatever it's collected so far. But we need to ensure that something is collected, right?
I realized about this when checking the [original metric_log stateless test](https://github.com/ClickHouse/ClickHouse/blob/31eeeae4fd8bf416715f3eb9e6a70bbeec4a5deb/tests/queries/0_stateless/00990_metric_log_table_not_empty.sql)

Reply:
Ok :)